CHANGELOG.md

```diff
@@ -54,9 +54,10 @@ The main changes of this release are:
 * Compute some memory/cpu Postgres tuning for the host [#335]
 * Provide a better pg_auto_failover upgrade experience [#296]
 * Add support for building against Postgres 13 [#312]
-
-#### Removed
-* The option --nodename is replaced by the option --hostname [#273]
+* Allow registering multiple standbys concurrently [#395]
+* Implement a retry policy for deadlocks and other transient failures [#359]
+* Implement support for interactive tmux sessions [#409]
```

Comment:
Maybe remove this line for now since it is behinde the PG_AUTOCTL_DEBUG=1 flag at the moment.

Reply:
Well this is documented at https://pg-auto-failover.readthedocs.io/en/latest/ref/reference.html#pg-autoctl-do. And I think an update with the `tmux` capabilities there would be good, now, in another PR.